components/exercises/chapter-5-oscar/Flattening/Flattening.tsx

```diff
@@ -0,0 +1,67 @@
+import { getFlattenedArray } from './getFlattenedArray';
+
+const arrays = [
+  ['❤️✨', '🚀🌟', '💭💪'],
+  ['🌅🔄', '💪😃'],
+  ['🚫🙅‍♂️', '💪🏆'],
+];
+
+export const phrases = [
```

Comment:
Same here, if this is named `phrases` I'm expecting the array to have the shape `['Believe in youself', 'Success is a journey']. But this is instead a collection of objects which include icons + phrase. Can we rename to something more descriptive?